ntoskrnl/config/cmapi.c

```diff
@@ -2360,6 +2368,19 @@ CmpEnumerateOpenSubKeys(
         return 0;
     }
 
+    /*
+     * Get the hash index of the root KCB and that of the
+     * parent if present. We need these indices to make sure
+     * that we do not lock the child and parent KCBs twice as
+     * they were locked in NtUnloadKey2 during normal hive
+     * unloading with no exclusive registry locking.
+     */
+    KcbIndex = GET_HASH_INDEX(RootKcb->ConvKey);
+    if (RootKcb->ParentKcb)
+    {
+        ParentKcbIndex = GET_HASH_INDEX(RootKcb->ConvKey);
```

Comment:
This is getting the same hash as KcbIndex. Didn't you mean this? :
```suggestion
        ParentKcbIndex = GET_HASH_INDEX(RootKcb->ParentKcb->ConvKey);
```